src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RequestJudgementByAdmissionForSpecCuiCallbackHandler.java

```diff
@@ -130,4 +147,13 @@ private String setUpBody() {
                 + "<br>This case will now proceed offline. Any updates will be sent by post.<br><br>"
         );
     }
+
+    private boolean isNonDivergent(CaseData caseData) {
+        return  MultiPartyScenario.isOneVOne(caseData)
+            || MultiPartyScenario.isTwoVOne(caseData)
```

Comment:
If you are checking only  if "(featureToggleService.isJudgmentOnlineLive()
            && isNonDivergent(data)
            && data.isLRvLipOneVOne()
            && data.isPayImmediately())" at line 106 then why you need all this logic? you are checking here 1v2 scenarios also then only doing it for 1v1 scenario (LRvLip)